pyeudiw/tools/jwk.py

```diff
@@ -0,0 +1,53 @@
+from enum import Enum
+
+from cryptojwt.jwk.jwk import key_from_jwk_dict
+from cryptojwt.jwk.ec import new_ec_key
+from cryptojwt.jwk.rsa import new_rsa_key
+from cryptography.hazmat.primitives import serialization
+
+
+class KeyType(Enum):
+    EC = 1
+    RSA = 2
+
+
+class JWK():
+    def __init__(self, key=None, keyType: KeyType = KeyType.RSA, hash_func: str = 'SHA-256') -> None:
+        if key:
+            self.key = key
+        elif keyType == KeyType.EC:
+            self.key = new_ec_key("P-256")
```

Comment:
"P-256" should be an argument of this init and not a constant

Reply:
Resolved in [8916cdf](https://github.com/italia/eudi-wallet-it-python/pull/14/commits/8916cdfae90af26b5f24e67d458e92a66d4dc639)